codalab/model/bundle_model.py

```diff
@@ -628,6 +629,10 @@ def set_offline_bundle(self, bundle):
                 # The user deleted the bundle.
                 return False
 
+            # Delete row in worker_run
```

Comment:
Is it documented somewhere what the semantics of something being in worker_run is?  This seems like tricky logic, and would be good to document how one should reason about the flow.

Reply:
I don't recall seeing anything detailing worker_run

and to be honest I'm only making educated guesses to guide my changes